cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -540,17 +541,24 @@ func newInstancesLookup(cfg v4.Configuration) integration.InstancesLookup {
 }
 
 // configureLogFormat checks the config and sets the log format accordingly.
-func configureLogFormat(logFormat string) {
-	if logFormat == config.LogFormatJSON {
+func configureLogFormat(cfg *config.Config) {
```

Comment:
not critical but I would pass `cfg.Log` struct instead of whole cfg ☝️

Reply:
The problem is that if the user is using the old `LogFormat`option this won't be reflected in the new configuration. Nonetheless, I agree that we should use the new LogConfig option for all logging settings. I will add a new commit with the changes.